src/main/java/com/dawn/jat/illuminati/post/advice/PostExceptionAdvice.java

```diff
@@ -1,15 +1,32 @@
 package com.dawn.jat.illuminati.post.advice;
 
 import com.dawn.jat.illuminati.post.exception.PostNotFoundException;
+import com.dawn.jat.illuminati.post.exception.QueryListPostNotFoundException;
+import java.util.LinkedHashMap;
+import java.util.Map;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 
 @ControllerAdvice
 public class PostExceptionAdvice {
-    @ExceptionHandler(value = PostNotFoundException.class)
+    /**
+     * Throw Exception when the query Slug Unavailable.
+     */
+    @ExceptionHandler(PostNotFoundException.class)
     public ResponseEntity<Object> exception(PostNotFoundException exception) {
         return new ResponseEntity<>(exception.getMessage(), HttpStatus.NOT_FOUND);
     }
+
+    /**
+     * Throw Exception when get List Post Not Found.
+     */
+    @ExceptionHandler(QueryListPostNotFoundException.class)
+    public ResponseEntity<Map<String, Object>> exceptionListPost(
+            final QueryListPostNotFoundException exception) {
+        Map<String, Object> errorInfo = new LinkedHashMap<>();
+        errorInfo.put("errorMessage", exception.getMessageFailResp());
```

Comment:
Directly handle the error body here

Reply:
![image](https://user-images.githubusercontent.com/7389488/72858621-f72f1980-3cf3-11ea-9ed9-2fd918c6cbd3.png)

Updated

Reply:
![image](https://user-images.githubusercontent.com/7389488/72858660-1b8af600-3cf4-11ea-937e-356bbe63d2c4.png)

Updated

Reply:
I think better keep it in the format
```
{
    error: {
        code: "204",
        message: "Cannot...."    
    },
    success: false
}
```